.github/workflows/pyspell.yml

```diff
@@ -0,0 +1,23 @@
+# This is workflow for spell checking using PySpelling lib (https://pypi.org/project/pyspelling/)
+name: Spellcheck
+# Controls when the action will run.
+on:
```

Comment:
@cjeanner I wonder if we want to optimize this job to only run when we change README.md using the path filter.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

but this is a light job we can run on all changes too.

Reply:
it's now filtered on only .md files. Thanks for the headsup!